### DISCUSSION TITLE
Unable to get the youtube comments Getting [] empty list as an output

### DISCUSSION THREAD

**Discussion by Appu2701** [obsei/obsei#331] [Q&A]
import json
from obsei.source.youtube_scrapper import YoutubeScrapperConfig, YoutubeScrapperSource
from obsei.sink.base_sink import BaseSink

ytConfig = YoutubeScrapperConfig(
    video_url="https://www.youtube.com/watch?v=UV1ZF6pyhtM",
    max_comments=20,
    fetch_replies=False
)

ytSource = YoutubeScrapperSource()

response = ytSource.lookup(ytConfig)

print("Raw response:", response)

class ListSink(BaseSink):
    data: list = []

    def send_data(self, analyzer_response):
        self.data.extend(analyzer_response)

list_sink = ListSink()

list_sink.send_data(response)

print("Processed data:", list_sink.data)

response_json = json.dumps([comment.dict() for comment in list_sink.data], indent=4)

print(response_json)

# Output

07/22/2024 14:54:37 - WARNING - obsei.misc.youtube_reviews_scrapper -   Unable to set sorting
Raw response: []
Processed data: []
[]

**Comment by Appu2701** [comment]
Why I am getting the response as an empty list []

**Reply by lalitpagaria** [reply]
It seems YouTube scrapping is broken. https://github.com/obsei/obsei/blob/master/obsei/misc/youtube_reviews_scrapper.py
Please refer https://github.com/egbertbouman/youtube-comment-downloader for latest working code.
